apps/src/templates/rubrics/LearningGoal.jsx

```diff
@@ -79,6 +182,32 @@ export default function LearningGoal({
             </div>
           </div>
         )}
+        <div>
+          <label className={style.teacherFeedbackLabel}>
+            <span>Teacher Feedback</span>
```

Comment:
Can you add this to `en_us.json` so it can be translated? There's a little bit of documentation [here](https://github.com/code-dot-org/code-dot-org/tree/staging/i18n#apps) but let me know if you need any pointers!

Reply:
Also for the other strings in this file!